main.go

```diff
@@ -135,7 +136,7 @@ func main() {
 	pflag.BoolVar(&enableLeaderElection, "enable-leader-election", false,
 		"Enable leader election for controller manager. "+
 			"Enabling this will ensure there is only one active controller manager.")
-	pflag.BoolVar(&createRBACPermissions, "create-rbac-permissions", false, "Automatically create RBAC permissions needed by the processors")
+	pflag.BoolVar(&createRBACPermissions, "create-rbac-permissions", false, "Automatically create RBAC permissions needed by the processors (deprecated)")
```

Comment:
I thought we wanted to keep this flag, just set the default to true eventually, and only create the RBAC resources if we have permission *and* the flag is set. This way, users who don't want the operator to do this can easily disable it.

Reply:
My understanding from the SIG call (as I reflected in #2588) was that we wanted to just check if the SA has the correct permissions. If users want to disable this feature, they will just remove the permissions from the SA (there is no other reason to keep those permissions otherwise, right?)

Reply:
I don't actually recall that discussion :sweat_smile: , but that sounds fair enough as long as we output a log about what the operator is doing. We should also make this clear in the documentation, so users don't need to hunt around the code to understand the operator's behaviour.